site/usage/webcomponent.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
I'm not sure whether we want the web component docs as part of the Vega-Lite docs. I think a link to the github page would be better. I defer to @kanitw and @arvind for a decision.

Reply:
I agree — the documentation pages should be reserved for first-party purposes. Documentation sets up an expectation of support for users and, while I appreciate the vega-element contribution, we don’t have the bandwidth to bring it under our umbrella officially.